README.md

```diff
@@ -10,10 +12,30 @@
 
 4. Are there unexpected regions with higher stars (rare patterns)? Something non-trivial that may be discovered are "hidden gems" such as food cart street or local spots
 
+5. Are more active yelpers prone to giving higher ratings or more thorough reviews?
+
+6. Do some yelpers give generally negative/positive reviews?
+
+7. Are there common words present in reviews with similar ratings?
+
+8. How consistent are ratings with common keywords?
+
+9. Do business ratings change over time? Can this change be attributed to attributes in the review data?
+
+10. What is the highest rated type of food (i.e. what restaurant is likely to please most?)
+
+11. What type of business receives the best/worst ratings?
```

Comment:
Both of these are good, but I think the more specific they are the better. For example, it would be interesting if yelp reviews could settle the age-long debate of:

- "Which city has the best X?" (steaks, ice cream, sandwich, ribs, seafood, etc.)

Reply:
Agree. I like questions 10 to 12, but I think we should remove them in favor of working more with the review text data.

Reply:
More concrete goals is better, agreed! I'll shift my thinking towards review text as the main focus. Numeric reviews may be too simple and not "interesting".